model/Database.php

```diff
@@ -359,6 +364,33 @@ function getUserByEmail($user_email)
         return $statement->fetch(PDO::FETCH_ASSOC);
     }
 
+    /**
+     * Get id and email for all admins
+     *
+     * @return array admin id and email data
+     * @author Keller Flint
+     */
+    function getAdmins()
+    {
+        $sql = "SELECT user_id, user_email FROM tutors.User WHERE user_is_admin = 1";
```

Comment:
Shouldn't this just say Users and not tutors.Users. If it is correct would you mind explaining it how it works?

Reply:
tutors is the database, Users is the table. You use this when you have to disambiguate tables in multiple databases in MySQL. I wrote it out of habit, it's redundant and I'm taking it out.